### DISCUSSION TITLE
Sending mutations to external services like Service Bus

### DISCUSSION THREAD

**Discussion by wario8a** [Azure/data-api-builder#2289] [General]
Is there any way to send a mutation as an event to an external service like a service bus message in that way we can integrate multiple process.

Thanks

**Comment by abhishekkumams** [comment]
@wario8a, can you give some more details for:
1. At what point in the mutation process do you want to send the event (before mutation execution, after successful execution, etc.)?
2. And, what information do you need to include in the event sent to the service bus?

**Reply by wario8a** [reply]
@abhishekkumams Thank you for you response,

1. Initially after the mutation, but it will be useful if we can choose the point to send the message. 
2. All fields, maybe the if possible some more information related with userid, date, etc.

**Comment by baskarkanini** [comment]
you can do like this.
 Create a class library for sending message.
 use this in the SSMS server and configure as unsafe dll in the SSMS. CLR enable to true.
 call this library via Stored procedure.
 
 Configure Stored procedure in your DAB. Send the parameters like queue /topic name, message from your DAB. Read these configurations from a configuration tables.
